.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+            though it doesn't prefix section labels with anything.)
+        -   Don't show the R code used to load the image, just its output (i.e., the image).
+        -   Give the figure the specified caption.
+    -   Use `insert_graphic("figures/stem/filename.ext")` to include your image. If you are using a `.pdf` image, make sure that you have the same file but as a `.svg` as well. 
+
+1.  Use `@Name1234` to refer to bibliography entries.
+
+1.  Use `\@ref(label)` to refer to labels for sections and figures.
+    Note that:
+    -   This only inserts the section number, not the words `Chapter`, `Appendix`, `Section`, or `Figure`.
+    -   Those are round parentheses, not curly braces.
+    -   If the figure's chunk ID is `stem-label`, use `fig:stem-label` to refer to it.
+
+1.  Glossary entries are in `gloss.md`, which is plain Markdown rather than R Markdown.
+    -   To refer to a glossary entry, use a direct link of the form `[text]``(glossary.html#term-label)`.
+    -   Glossary definitions are set in bold and use an HTML anchor tag to provide the ID.
+        We should find a more elegant way to do this.
+
+1.  Many code fragments are accompanied by an HTML comment that contains `src="stem/filename.ext"`.
+    This identifies the path to that file within the `./src` folder.
+    We should find a more elegant way to handle these references.
+
+1.  There are also some HTML comments containing the word `noindent` left over from formatting with an earlier template.
+    These were used to prevent indentation of the first line of continuation paragraphs.
+    We will find a more elegant way to handle this as we get closer to production.
+    
+    -   Do *not* use `View` in your R snippets,
+        as Travis will fail when it tries to launch a viewer from the command line.
+    
+
+## Using Python Code Chunks
+
+R users should be already familiar with using R Markdown. The process for using
+Python in R Markdown is familiar straight forward. When you want to include some
+code and its output, use code chunks like:
+    
+    ```{python}
+    python code ...
+    ```
+
+Code chunks have
+[options](https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf)
+that apply to both R and Python.
+    
+## When Using Other Software Packages
+
+Travis needs to know what packages to install to build the website:
+
+- For R packages, it is very easy. When in the Merely Useful R Project (by
+opening the `merely-useful.github.io.Rproj` file), type
+`usethis::use_package("packagename")` in the R Console. This will add the
+package dependency to the `DESCRIPTION` file under the imports section.
+
+- For Python packages you'll need to add the package to the `.travis.yml` under 
+the `addons` section:
+
+```yaml
+addons:
+  apt:
+    packages:
+    - python3
+    # Add more packages below here
+    # and looks like "python3-packagename"
+    # e.g.:
+    - python3-numpy
+```
+
+## Guide for code and text styling
```

Comment:
Maybe move this above the previous section and then move down the Travis section I commented on above to the bottom. Then both the travis sections are at the end.